tests/e2e/testsLibrary/WorkspaceHandlingTests.ts

```diff
@@ -29,16 +30,37 @@ export class WorkspaceHandlingTests {
         @inject(CLASSES.Dashboard) private readonly dashboard: Dashboard,
         @inject(CLASSES.CreateWorkspace) private readonly createWorkspace: CreateWorkspace,
         @inject(CLASSES.Workspaces) private readonly workspaces: Workspaces,
-        @inject(CLASSES.WorkspaceNameHandler) private readonly workspaceNameHandler: WorkspaceNameHandler) {}
+        @inject(CLASSES.BrowserTabsUtil) private readonly browserTabsUtil: BrowserTabsUtil) {}
+
+    public getWorkSpaceName(stack: string) {
```

Comment:
@makambalaji  This doesn't look like a safe method. will make it super confusing when the project name changes or new devfiles are added. please revert this `switch` case

Reply:
Not using this code, any more.. Alternative is already added,

Reply:
could you please remove the redundant code then?

Reply:
Removed the redundant code